daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector-jaeger.md

```diff
@@ -77,15 +85,20 @@ Jaeger V2 can be deployed using the OpenTelemetry Operator for simplified manage
 
 #### Installation
 
-1. **Install cert-manager** to manage certificates:
-   ```bash
-   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.19.1/cert-manager.yaml -n cert-manager
-   ```
-   Verify that all resources in the `cert-manager` namespace are ready.
+> **Note:**In order for the API server to communicate with the webhook component of the operator, the webhook requires a TLS certificate that the API server is configured to trust. There are a few different ways you can use to generate/configure the required TLS certificate detailed in the [otel operator chart docs](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#tls-certificate-requirement)
+
+For simplicity in this case we will be letting helm to create an automatically generated self-signed certificate,
+ 
+```bash
```

Comment:
Did you mean to add something here in this bash statement, or is this redundant?